src/lib/apparmor/profiles.rb

```diff
@@ -0,0 +1,192 @@
+# Get the status of profiles loaded
+#  - enforced
+#  - complain
+# Uses aa-status --json
+
+require 'json'
+require 'open3'
+Yast.import 'UI'
+Yast.import 'Label'
+Yast.import 'Popup'
+
+module AppArmor
+  # Class representing a single apparmor profile
+  class Profile
+    attr_reader :name, :status, :pid
+
+    def initialize(name, status)
+      textdomain 'apparmor'
+      @name = name
+      @status = status
+      @pid = []
+    end
+
+    # Return/Set the name of the profile
+    def name(n = '')
+      @name = n unless n.empty?
+      @name
+    end
```

Comment:
It turns out, I am not using any of these functions as setter or a getter. Removed it.